framework/update/internal/appupdateservice.cpp

```
@@ -455,8 +554,64 @@ void AppUpdateService::downloadPreviousReleasesNotes(const Version& updateVersio
 void AppUpdateService::clear()
 {
     m_lastCheckResult = RetVal<ReleaseInfo>::make_ok(ReleaseInfo());
+}
+
+void AppUpdateService::cleanupStalePackages(const std::string& keepFileName)
```

Clang (14.0.6)

[warning] method 'cleanupStalePackages' can be made static

(readability-convert-member-functions-to-static)